clients/cli/src/main/java/org/apache/gravitino/cli/FullName.java

```diff
@@ -29,6 +29,8 @@ public class FullName {
   private final CommandLine line;
   private String metalakeEnv;
   private boolean matalakeSet = false;
+  private boolean isMissingNameInfoVisible = true;
+  private boolean isMalformedInfoVisible = true;
```

Comment:
These names are inverted. isMissingNameInfoVisible is true when the missing name message has not been displayed.

Reply:
@justinmclean 
- `hasDisplayMissingNameInfo`: When the variable is true, it has been displayed once and no more error messages will be displayed 
- `hasDisplayMalformedInfo`: When the variable is true, it has been displayed once and no more error messages will be displayed
Is this more reasonable？

Reply:
They are better but probably should be "hasDisplayed..." or "hasShown..."

Reply:
fix